libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -623,14 +623,24 @@ def _response_to_result(
         input_tokens = response.usage_metadata.prompt_token_count
         output_tokens = response.usage_metadata.candidates_token_count
         total_tokens = response.usage_metadata.total_token_count
+        thought_tokens = response.usage_metadata.thoughts_token_count
         cache_read_tokens = response.usage_metadata.cached_content_token_count
         if input_tokens + output_tokens + cache_read_tokens + total_tokens > 0:
-            lc_usage = UsageMetadata(
-                input_tokens=input_tokens - prev_input_tokens,
-                output_tokens=output_tokens - prev_output_tokens,
-                total_tokens=total_tokens - prev_total_tokens,
-                input_token_details={"cache_read": cache_read_tokens},
-            )
+            if thought_tokens > 0:
```

Comment:
what is it's None?

Reply:
the value from the proto should be 0 if it is unset.